docs/general/builtins/scalar-functions.rst

```diff
@@ -2446,6 +2446,46 @@ null_string
 If the ``null_string`` argument is omitted or NULL, none of the substrings of
 the input will be replaced by NULL.
 
+.. _scalar-array-min:
+
+``array_min(anyarray)``
+---------------------------------------------------------
+
+The ``array_min`` function finds minimal element in a given array.
+Return value is always of the type of input array.
+If array argument is ``NULL``, the result is ``NULL``.
+If array argument is empty array, the result is ``NULL``.
```

Comment:
```suggestion
The ``array_min`` function returns the smallest element in ``array``. If
``array`` is ``NULL`` or an empty array, the function returns``NULL``.
```
not sure what this line means:

> Return value is always of the type of input array.

I presume it means the same type as the smallest element? if it does mean that, then I think that is implied and doesn't need stating. we typically only explicitly mention it if a type cast _is_ performed